content/en/docs/reference/glossary/eviction.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Eviction
+id: eviction
+date: 2021-05-08
+full_link: /docs/concepts/scheduling-eviction/
+short_description: >
+    Process of terminating one or more Pods on Nodes

```

Comment:
great job! Definition is correct, content renders perfectly and links are valid.